requirements_test_all.txt

```diff
@@ -982,6 +982,9 @@ pykulersky==0.5.2
 # homeassistant.components.lastfm
 pylast==4.2.1
 
+# homeassistant.components.launch_library
+pylaunches==1.2.0
```

Comment:
Is this correct, this was added when i ran VSCode task, `generate requirements`?

Reply:
yes